sdk/src/wallet/account/operations/output_claiming.rs

```diff
@@ -50,88 +50,63 @@ impl Account {
         let mut output_ids_to_claim: HashSet<OutputId> = HashSet::new();
         for (output_id, output_data) in &account_details.unspent_outputs {
             // Don't use outputs that are locked for other transactions
-            if !account_details.locked_outputs.contains(output_id) {
-                if let Some(output) = account_details.outputs.get(output_id) {
-                    match &output.output {
-                        Output::Basic(basic_output) => {
```

Comment:
Is it correct to remove Basic/Nft match? Isn't it going to pick aliases and foundries?

Reply:
Good catch, still needs to be handled

Reply:
Maybe you can simply filter the iterator by `is_basic || is_nft` @Alexandcoats